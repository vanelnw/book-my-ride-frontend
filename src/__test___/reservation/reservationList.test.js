import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import { Provider } from 'react-redux';
import store from '../../redux/store';
import ReservationList from '../../components/reservation/ReservationList';

it('renders correctly', () => {
  const tree = renderer
    .create(
      <Provider store={store}>
        <Router>
          <ReservationList />
        </Router>
      </Provider>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
