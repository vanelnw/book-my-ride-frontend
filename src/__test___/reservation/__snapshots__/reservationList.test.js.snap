// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="my-reservations-container"
>
  <div
    className="heading"
  >
    <h1>
      My Reservations
    </h1>
  </div>
</div>
`;
